src/app/main-page/main-page.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-main-page',
+  templateUrl: './main-page.component.html',
+  styleUrls: ['./main-page.component.scss']
+})
+export class MainPageComponent {
+
+  constructor(private router: Router){}
+
+  PokemonProject() :void{
+    this.router.navigate(['/login']);
```

Comment:
please try to think how you can union between the two function ? 
instead of creating two function create one function of navigation.

Reply:
fixed